analysis_driver/quality_control/relatedness.py

```diff
@@ -168,7 +172,7 @@ def _run(self):
             job_name='gatk_genotype_gvcfs',
             working_dir=self.job_dir,
             cpus=12,
-            mem=96
+            mem=self.memory
```

Comment:
make sure the memory passed to slurm is slightly greater than the memory passed to java (+2G maybe)